components/Helpers.php

```diff
@@ -513,7 +513,7 @@ public static function helper( $params, $obj = null ) {
 		$out = ob_get_clean();
 
 		$out = apply_filters( 'pods_helpers_post_helper', $out, $params, $helper );
-		$out = apply_filters( 'pods_helpers_post_helper_' . $helper['slug'], $out, $params, $helper );
+		$out = apply_filters( "pods_helpers_post_helper_{$helper['slug']}", $out, $params, $helper );
```

Comment:
Doesn't fit the same way the others were changed, can you set as var and then put in filter name?

Reply:
Addressed in a8f4ee864afea764b8ba67c836dc1c675c1f5582.